frontend/public/locales/en/translation.json

```diff
@@ -2294,7 +2294,7 @@
     "template.emptyStateMsg": "Click <bold>Create Ansible template</bold> to create your resource.",
     "template.hint": "Create an Ansible automation job template to automatically run Ansible jobs at different stages of the cluster life cycle. To use this feature, the Ansible Automation Platform Resource Operator must be installed. Install the operator through the following link:",
     "template.hint.noConsoleURL": "Create an Ansible automation job template to automatically run Ansible jobs at different stages of the cluster life cycle. To use this feature, the Ansible Automation Platform Resource Operator must be installed. Install the operator through OperatorHub.",
-    "template.information.description": "The default job templates that you select appear automatically during cluster creation. To create a sequence of events, select multiple jobs. Drag and drop the job templates to reorder the sequence.",
+    "template.information.description": "To create a sequence of events, add multiple jobs. Drag and drop the job templates to reorder the sequence.",
```

Comment:
First sentence did not convey anything meaningful. And the rest of the text applies better to the Install and Upgrade steps.